<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>49</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="siui::VolumeSpinBox" name="volumeSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="volume" stdset="0">
      <double>12.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>at</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="siui::PressureSpinBox" name="pressureSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="unit" stdset="0">
      <enum>siui::PressureSpinBox::hectopascal</enum>
     </property>
     <property name="pressure" stdset="0">
      <double>14.000000000000000</double>
     </property>
     <property name="prefix">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>represent</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="siui::EnergyLabel" name="energyLabel"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>siui::EnergyLabel</class>
   <extends>QLabel</extends>
   <header>siui/EnergyLabel.hpp</header>
  </customwidget>
  <customwidget>
   <class>siui::PressureSpinBox</class>
   <extends>QWidget</extends>
   <header>siui/PressureSpinBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>siui::VolumeSpinBox</class>
   <extends>QWidget</extends>
   <header>siui/VolumeSpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
